server/src/session.ts

```diff
@@ -201,6 +201,23 @@ export class Session {
     project.getLanguageService().getSemanticDiagnostics(fileName);
   }
 
+  private suggestStrictMode(project: ts.server.Project) {
```

Comment:
```suggestion
  private handleCompilerOptionsDiagnostics(project: ts.server.Project) {
```
What do you think of making this function name a bit more generic?